views/stats-proofofplay-page.twig

```diff
@@ -69,6 +69,60 @@
 
 {% block javaScript %}
     <script type="text/javascript">
+
+        function statsExportFormSelect(dialog) {
+
+            var fromDtLink;
+            var toDtLink;
+            var displayId;
+
+            $(dialog).on("change", function() {
+                fromDtLink =  $(dialog).find("#fromDt").val();
+                toDtLink =  $(dialog).find("#toDt").val();
+                displayId =  $(dialog).find("#displayId").val();
+
+                if (
+                     fromDtLink === null ||  toDtLink === null  ||
+                     fromDtLink === '' ||  toDtLink === ''  ||
+                     fromDtLink === undefined || toDtLink === undefined
+                ) {
+
+                    //
+
+                } else {
+
+                    $(dialog).find(".total-stat").remove();
+                    $(dialog).find('.save-button').prop('disabled', true);
+                    $(dialog).find(".loading-overlay").show();
+
+                    $.ajax({
+                        type: "get",
+                        url: "{{ urlFor("stats.export") }}",
```

Comment:
As with prior comment, I think this should be a separate route so that the request is "restful" and only does one thing